packages/tabs/src/tab-bar.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    class="el-tabs__active-bar"
+    :class="`is-${ rootTabs.props.tabPosition }`"
+    :style="barStyle"
+  ></div>
+</template>
+<script lang='ts'>
+import { defineComponent, inject, getCurrentInstance, watch, nextTick, ref } from 'vue'
+import { firstUpperCase } from '@element-plus/utils/util'
+
+export default defineComponent({
+  name: 'ElTabBar',
+  props: {
+    tabs: {
+      type: Array,
+      default: () => [],
```

Comment:
Specify the type here using 
```suggestion
type: Array as PropType<TypeAsItIs[]>
default: () => [] as TypeAsItIs[]
```